src/Components/v2/__stories__/CarouselV2.story.tsx

```diff
@@ -0,0 +1,186 @@
+              return (
+                <Image
+                  px={5}
+                  src={props.resized.url}
+                  width={props.resized.width}
+                  height={props.resized.height}
+                />
+              )
+            }}
+          />
+        </Box>
+      </Section>
+      <Section title="Large Carousel with WrapAround">
+        <Box width="70%">
+          <LargeCarousel
+            data={images}
+            height={300}
+            settings={{ wrapAround: true }}
+            render={props => {
+              return (
+                <Image
+                  px={5}
+                  src={props.resized.url}
+                  width={props.resized.width}
+                  height={props.resized.height}
+                />
+              )
+            }}
+          />
+        </Box>
+      </Section>
+      <Section title="Artwork Brick">
+        <Box width="70%">
+          <RelayStubProvider>
+            <Carousel
+              data={artworkBricks}
+              render={artwork => {
+                const {
+                  node: {
+                    image: { aspect_ratio },
+                  },
+                } = artwork
+
+                return (
+                  <FillwidthItem
+                    artwork={artwork.node}
+                    targetHeight={200}
+                    imageHeight={200}
+                    width={200 * aspect_ratio}
+                    margin={20}
+                  />
+                )
+              }}
+            />
+          </RelayStubProvider>
+        </Box>
+      </Section>
+      <Section title="Custom Arrows with defaults">
+        <Box width="70%">
+          <Carousel
+            data={images}
+            render={props => {
+              return (
+                <Image
+                  px={5}
+                  src={props.resized.url}
+                  width={props.resized.width}
+                  height={props.resized.height}
+                />
+              )
+            }}
+            renderLeftArrow={({ Arrow }) => {
+              return (
+                <Box top={10} position="relative" bg="black10">
+                  <Arrow />
+                </Box>
+              )
+            }}
+            renderRightArrow={({ Arrow }) => {
+              return (
+                <Box top={10} position="relative" bg="black10">
+                  <Arrow />
+                </Box>
+              )
+            }}
+          />
+        </Box>
+      </Section>
+      <Section title="Custom Arrows ">
+        <Box width="70%">
+          <Carousel
+            data={images}
+            render={props => {
+              return (
+                <Image
+                  px={5}
+                  src={props.resized.url}
+                  width={props.resized.width}
+                  height={props.resized.height}
+                />
+              )
+            }}
+            renderLeftArrow={({ getFlickity }) => {
```

Comment:
Since this is no longer a function call or a getter, the API can be updated to simply `flickity`. 

Once this PR merged, and a second PR is opened swapping out the old carousel for the new (removing V2) will need to search reaction and update the API where these render callbacks are being used (I think just one spot).